bin/pycbc_condition_strain

```diff
@@ -75,6 +75,7 @@ parser.add_argument('--frame-duration', metavar='SECONDS', type=int,
                          'in seconds')
 
 pycbc.strain.insert_strain_option_group(parser)
+pycbc.psd.insert_psd_option_group(parser)
```

Comment:
Why do you have this? You don't even use the corresponding 'from_cli' function that would use the options in this group. Is this a leftover from some test?

Reply:
I remember the build test will call `pycbc_condition_strain`, if I don't add this, then `opt` in this file doesn't have `extra-args`.

Reply:
@WuShichao Take another pass through reasoning that through. Why did you get an error? What is the purpose of these option groups? If you never call its corresponding function that uses the options. How can it fail?